src/main/kotlin/me/zeroeightsix/kami/feature/module/Nuker.kt

```diff
@@ -38,6 +43,15 @@ object Nuker : Module() {
     private var progress = 0.0
     private var currentBlock: BlockPos? = null
 
+    private val range
+        get() =
+            // this doesn't look that great, but i can't think of a cleaner way to do this
```

Comment:
This doesn't look bad - it's fine.

Do remove the comment though. Next time just add a comment on your PR instead of in the code!